File: python/8-even-or-odd.py
def even_or_odd(n):
    return ["Even", "Odd"][int("{0:b}".format(1)) & 1]
    

# sol 1
def sol1(n):
    return ["Even", "Odd"][n % 2]
 

print(even_or_odd(2))
print(sol1(2))

File: go/8_even_or_odd.go
package main

func even_or_odd(n int) string {
	if n%2 == 0 {
		return "Even"
	}
	return "Odd"
}

// sol 1
func sol1(n int) string {
	// using bitwise & operator
	return []string{"Even", "Odd"}[n&1]
}

// input
func main() {
	println(even_or_odd(9))
	println(sol1(9))
}
